upgrade/upgrade-1.3.1.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+/**
+ * Update main function for module Version 1.3.1
+ *
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_1_3_1($module)
+{
+    return (new PrestaShop\Module\PrestashopCheckout\OrderStates())->installPaypalStates();
```

Comment:
No need to add `upgrade/upgrade-1.3.1.php`, action is already triggered by `upgrade/upgrade-1.3.0.php` and we have to make another beta for 1.3.0 so we will add this color changes 👍